### PR TITLE
Gardens dark mode support

### PR DESCRIPTION
**Pull request by 0xGabi**
The `@1hive/1hive-ui` was already updated with the changes in https://github.com/1Hive/1hive-ui/pull/22

With the latest changes we made to the Onboarding and a few places of the UI now we have extra work to support dark mode for Gardens.

Figma designs: 
- https://www.figma.com/file/qbI6hNV2u2tbfpqF2ZiEgK/🌑-Honey-Pot-Dark-Mode?node-id=63%3A6641
- https://www.figma.com/file/T0xI9YXz3MRJVyWGpT5mIs/🌚-Dark-mode-Gardens?node-id=0%3A1

TODO:

- [x] Fix the flickering issue on the first render.
- [x] Migrate Onboarding components to use dark mode.
- [x] Edit DiscourseComments.css to be compatible with both light and dark themes.
- [x] Fix `theme.negativeSurface` on infoDatas.
- [x] Migrate `utils/theme.js` to `@1hive/1hive-ui`.
- [x] Import connect-icon for dark-mode on Staking component.
- [x] Fix `disable` styles

### CODE REVIEW

**Comment by Corantin** [comment]
Just to notice that there is a 10HNY rewarded Quest for having a dark theme in Gardens

File: src/components/Garden/Feed/HeroBanner.js [hunk 1]
```diff
@@ -22,15 +28,29 @@ const BANNERS = {
...
+  const { appearance } = useAppTheme()
 
   const mobileMode = layoutName === 'small'
   const tabletMode = layoutName === 'medium'
   const compactMode = mobileMode || tabletMode
+
+  const bannerType = appearance === 'light' ? BANNERS : BANNERS_DARK_MODE
+
   const banner =
-    fullScreenMode && tabletMode ? BANNERS.medium_full : BANNERS[layoutName]
```
Comment (Mati0x) [inline src/components/Garden/Feed/HeroBanner.js:33]: No esta en Staging el fullScreenmMode

Comment (pavloMandryk) [reply src/components/Garden/Feed/HeroBanner.js:33]: Seems like something we added for handling [some error on no byot orgs](https://github.com/1Hive/gardens-ui/commit/3ec12cdeaf7c6032e1ef7177149570d12c72afd6)

Currently it is not implemented on staging.

**Review by 0xGabi — COMMENTED** [review]
LGTM 🌝

**Review by Mati0x — APPROVED** [review]
Go throw all files changes and everything seems right. The app in localhost and vercel works fine and the UI in dark mode is pretty awesome

Ready to go!

**Pull request merged**